### PR TITLE
Update ACL example

### PR DESCRIPTION
**Pull request by 0xdavinchee**
* update ethereum-contracts to v1.2.2
* utilize ERC1820RegistryCompiled
* fix SFDeployer to take new syntax
* fix Makefile commands

per https://superfluidhq.slack.com/archives/C02G67JCG06/p1652118458252829?thread_ts=1652118444.875529&cid=C02G67JCG06

### CODE REVIEW

File: examples/simple-acl-close/src/test/SimpleACLCloseResolver.t.sol [hunk 1]
```diff
@@ -47,15 +48,14 @@ contract SimpleACLCloseResolverTest is Test {
...
 
         sfDeployer = new SuperfluidFrameworkDeployer();
-        (_host, _cfa, , _superTokenFactory) = sfDeployer.getFramework();
+        SuperfluidFrameworkDeployer.Framework memory sfFramework = sfDeployer
+            .getFramework();
+        _host = sfFramework.host;
+        _cfa = sfFramework.cfa;
+        _superTokenFactory = sfFramework.superTokenFactory;
         // Deploy and retrieve contracts using `_vm` and `sender`. The sender deploys everything.
         _cfaLib.host = _host;
```
Comment (hellwolf) [inline examples/simple-acl-close/src/test/SimpleACLCloseResolver.t.sol]: you can use sf.cfaLib directly.

**Review by hellwolf — CHANGES_REQUESTED** [review]

File: examples/simple-acl-close/src/test/SimpleACLCloseResolver.t.sol (hunk 1 above)
Comment (0xdavinchee) [reply examples/simple-acl-close/src/test/SimpleACLCloseResolver.t.sol]: oh true 😅

Comment (0xdavinchee) [reply examples/simple-acl-close/src/test/SimpleACLCloseResolver.t.sol]: fixed

**Review by hellwolf — APPROVED** [review]
https://xkcd.com/799/

image

**Pull request merged**